conf/orchestrator-simple.conf.json

```diff
@@ -3,38 +3,58 @@
   "#": "this is a comment. See further comment hints below.",
   "#": "-----",
   "Debug": true,
+  "EnableSyslog": false,
   "ListenAddress": ":3000",
   "MySQLTopologyUser": "orc_client_user",
   "MySQLTopologyPassword": "orc_client_password",
+  "MySQLTopologyCredentialsConfigFile": "",
+  "MySQLTopologySSLPrivateKeyFile": "",
+  "MySQLTopologySSLCertFile": "",
+  "MySQLTopologySSLCAFile": "",
+  "MySQLTopologySSLSkipVerify": true,
+  "MySQLTopologyUseMutualTLS": false,
   "MySQLOrchestratorHost": "127.0.0.1",
   "MySQLOrchestratorPort": 3306,
   "MySQLOrchestratorDatabase": "orchestrator",
   "MySQLOrchestratorUser": "orc_server_user",
   "MySQLOrchestratorPassword": "orc_server_password",
+  "MySQLOrchestratorCredentialsConfigFile": "",
+  "MySQLOrchestratorSSLPrivateKeyFile": "",
+  "MySQLOrchestratorSSLCertFile": "",
+  "MySQLOrchestratorSSLCAFile": "",
+  "MySQLOrchestratorSSLSkipVerify": true,
+  "MySQLOrchestratorUseMutualTLS": false,
   "#": "-----",
   "#": "The above is all you need for initial setup. Below is more advanced stuff",
   "#": "-----",
   "MySQLConnectTimeoutSeconds": 1,
+  "DefaultInstancePort": 3306,
   "DiscoverByShowSlaveHosts": true,
   "InstancePollSeconds": 5,
   "DiscoveryIgnoreReplicaHostnameFilters": [
     "a_host_i_want_to_ignore[.]example[.]com",
     ".*[.]ignore_all_hosts_from_this_domain[.]example[.]com",
     "a_host_with_extra_port_i_want_to_ignore[.]example[.]com:3307"
   ],
+  "UnseenInstanceForgetHours": 240,
+  "SnapshotTopologiesIntervalHours": 0,
+  "InstanceBulkOperationsWaitTimeoutSeconds": 10,
   "HostnameResolveMethod": "default",
   "MySQLHostnameResolveMethod": "@@hostname",
+  "SkipBinlogServerUnresolveCheck": true,
+  "ExpiryHostnameResolvesMinutes": 60,
+  "RejectHostnameResolvePattern": "",
   "ReasonableReplicationLagSeconds": 10,
   "ProblemIgnoreHostnameFilters": [],
+  "VerifyReplicationFilters": false,
+  "ReasonableMaintenanceReplicationLagSeconds": 20,
+  "VerifyReplicationFilters": false,
+  "ReasonableMaintenanceReplicationLagSeconds": 20,
+  "CandidateInstanceExpireMinutes": 60,
```

Comment:
these lines are duplicated?

Reply:
@EagleEyeJohn good catch. Would either you or @jfg956 be interested to submit a pull request to fix this?